internal/rpc/conversation/conversaion.go

```diff
@@ -402,6 +402,14 @@ func (c *conversationServer) SetConversationMaxSeq(ctx context.Context, req *pbc
 	return &pbconversation.SetConversationMaxSeqResp{}, nil
 }
 
+func (c *conversationServer) SetConversationMinSeq(ctx context.Context, req *pbconversation.SetConversationMinSeqReq) (*pbconversation.SetConversationMinSeqResp, error) {
+	if err := c.conversationDatabase.UpdateUsersConversationField(ctx, req.OwnerUserID, req.ConversationID,
```

Comment:
This method is update `conversation` collection. but Seq is save in `seq` collection in DB.

Need check.

@FGadvancer